conformance/src/test/scala/za/co/absa/enceladus/conformance/interpreter/rules/CastingRuleSuite.scala

```diff
@@ -51,16 +51,14 @@ class CastingRuleSuite extends FunSuite with SparkTestBase {
     val conformedJSON = JsonUtils.prettySparkJSON(conformed.orderBy($"id").toJSON.collect)
 
     if (conformedJSON != CastingRuleSamples.conformedOrdersJSON) {
-      println("EXPECTED:")
-      println(CastingRuleSamples.conformedOrdersJSON)
-      println("ACTUAL:")
-      println(conformedJSON)
-      println("DETAILS (Input):")
-      inputDf.printSchema()
-      inputDf.show
-      println("DETAILS (Conformed):")
-      conformed.printSchema()
-      conformed.show
+      logger.debug("EXPECTED:")
```

Comment:
These should probably be `.error`. I see you've already done that in [previous PRs].(https://github.com/AbsaOSS/enceladus/pull/728/files#diff-dc8efb232378bd7dcc7b766d4c264cb1R1020)